### DISCUSSION TITLE
Endlessly in the logs: "W (908245) WIFI: Disconnected (201, No AP)"

### DISCUSSION THREAD

**Discussion by KruseLuds** [jomjol/AI-on-the-edge-device#3066] [Q&A] [answered]
I am stuck with this forever in the logs:

All I can access are the logs, when using the "connect" button on this page: https://jomjol.github.io/AI-on-the-edge-device/

192.168.4.1 has never workled and the IP address I have assigned the device in it's win.ini is also unreachable - always. I have a very strong wifi network with numerous AP's (each connected by CAT6 to the router) and I am sitting literally 5 feet away from one of the access points...

```
W (869595) WIFI: Disconnected (201, No AP)
W (872005) WIFI: Disconnected (201, No AP)
W (874425) WIFI: Disconnected (201, No AP)
W (876835) WIFI: Disconnected (201, No AP)
E (876845) WIFI: Disconnected, multiple reconnect attempts failed (201), still retrying...
W (879255) WIFI: Disconnected (201, No AP)
W (881665) WIFI: Disconnected (201, No AP)
W (884085) WIFI: Disconnected (201, No AP)
W (886505) WIFI: Disconnected (201, No AP)
W (888915) WIFI: Disconnected (201, No AP)
W (891335) WIFI: Disconnected (201, No AP)
W (893745) WIFI: Disconnected (201, No AP)
W (896165) WIFI: Disconnected (201, No AP)
W (898575) WIFI: Disconnected (201, No AP)
W (900995) WIFI: Disconnected (201, No AP)
E (900995) WIFI: Disconnected, multiple reconnect attempts failed (201), still retrying...
W (903405) WIFI: Disconnected (201, No AP)
W (905825) WIFI: Disconnected (201, No AP)
W (908245) WIFI: Disconnected (201, No AP)
W (910655) WIFI: Disconnected (201, No AP)
```

Any ideas anybody might have would be very helpful - Note, I have also shown more logs [here](https://github.com/jomjol/AI-on-the-edge-device/discussions/2955#discussioncomment-9336584) as well.

Thank you in advance

**Comment by caco3** [comment]
> 192.168.4.1 has never workled 

Why did this not work for you?
What was the log there?
Maybe your antenna is bad?

**Reply by KruseLuds** [reply, marked as answer]
I have resolved the issue, did alot of digging, it worked on my regular internal home network but did not on my IOT vlan. It turns out I had 802.11r turned on one but not for the IOT. Turns that on and all is well. Next step is to set up the MQTT to get it into Home Assistant. I have opther questions pbut will post them under a different discussion with the correct title/s!

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/96640689/1742d2b8-8a38-4508-b65d-b26778026561)
